tests/e2e/specs/shopper/mini-cart-drawer.test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+
+const block = {
+	name: 'Mini Cart Block',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Mini Cart → Can open/close the drawer', () => {
```

Comment:
I'm wondering… 🤔  would it make sense structurally to have a high level `describe` block for the whole `Mini Cart` thing and then nest the functionalities within other `describe`s?

So, it would look something like this:

```js
describe( 'Mini Cart', () => {
  describe( 'Drawer', () => {
    it( 'should open when shopper clicks outside', () => { /* ... */ } )
  } );

  describe( 'Icon', () => {
    it( 'should be visible with a badge on the front-end', () => { /* ... */ } )
  } );
} );
```

Notice how this reads much more like smooth English and already organizes the tests for you. And in the console it would be neatly printed as well, which is a plus.

What do you think?

Reply:
@sunyatasattva I think we have the winner here:

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/5423135/155261477-7c36bab6-3826-4a6e-b79e-291a51cbc923.png">
	
<img width="641" alt="image" src="https://user-images.githubusercontent.com/5423135/155261584-9c7fd428-c06e-41a3-b668-51806473adc2.png">

Reply:
I thought splitting the test suites into separated files is easier to maintain and run (Eg: running a single test case using a file filter). But I don't think we need to run a single test suite very option. And we still can run a single test suite using filters.

Reply:
The problem with this approach is the test file can be very long. Combining Icon and Drawer tests, the test file is 100+ lines. It's not a problem for me but I'm not sure about others' preferences.